 include/local.h | 2 +-
 src/confmisc.c  | 3 ++-
 2 files changed, 3 insertions(+), 2 deletions(-)

diff --git a/include/local.h b/include/local.h
index 512e4455..42ffddea 100644
--- a/include/local.h
+++ b/include/local.h
@@ -281,7 +281,7 @@ extern snd_lib_error_handler_t snd_err_msg;
  */
 #define HAVE_GNU_LD
 #define HAVE_ELF
-#define HAVE_ASM_PREVIOUS_DIRECTIVE
+//#define HAVE_ASM_PREVIOUS_DIRECTIVE
 
 /* Stolen from libc-symbols.h in GNU glibc */
 
diff --git a/src/confmisc.c b/src/confmisc.c
index 9b30d6c1..592d0113 100644
--- a/src/confmisc.c
+++ b/src/confmisc.c
@@ -80,7 +80,8 @@
 #include <string.h>
 #include <ctype.h>
 #include <limits.h>
-
+// __out conflicts with macros in Windows SDK
+#undef __out
 /**
  * \brief Gets the boolean value from the given ASCII string.
  * \param ascii The string to be parsed.