 shell/hush.c                     |   4 +-
 1 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/shell/hush.c b/shell/hush.c
index e6be70078..a56eb7000 100644
--- a/shell/hush.c
+++ b/shell/hush.c
@@ -350,9 +350,9 @@
 #endif
 #include <glob.h>
 /* #include <dmalloc.h> */
-#if ENABLE_HUSH_CASE
+//#if ENABLE_HUSH_CASE
 # include <fnmatch.h>
-#endif
+//#endif
 #include <sys/times.h>
 #include <sys/utsname.h> /* for setting $HOSTNAME */
 